src/content/en/tools/chrome-devtools/network-performance/index.md

```diff
@@ -0,0 +1,257 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Get started analyzing network performance.
+
+{# wf_updated_on: 2016-12-15 #}
+{# wf_published_on: 2016-12-15 #}
+
+<style>
+.devtools-inline {
+  max-height: 1em;
+  vertical-align: middle;
+}
+figcaption {
+  text-align: center;
+}
+</style>
+
+# Get Started with Analyzing Network Performance in Chrome DevTools {: .page-title }
+
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+In this step-by-step, interactive tutorial, you learn how to use the Chrome
+DevTools Network panel to understand why a particular web page is loading
+slowly. The general workflow that you learn is helpful for analyzing network
+performance on any web page.
+
+## Step 1: Set up DevTools {: #set-up }
+
+Suppose that you're receiving reports from mobile users that a particular web
+page is slow. Your job is to make the page fast.
+
+Follow along with the instructions below to open DevTools on the page that
+your users are reporting is slow.
+
+1. Click **Open Slow Page**. The page opens in a new tab.
+
+     <a href="https://googlechrome.github.io/devtools-samples/network/gs/v1.html"
+       target="devtools" class="gc-analytics-event" rel="noopener noreferrer"
+       data-category="DevTools / Network / Get Started"
+       data-label="Slow Page Opened">
+       <button>Open Slow Page</button>
+     </a>
+
+1. While the page is in focus, press
+   <kbd>Command</kbd>+<kbd>Shift</kbd>+<kbd>I</kbd> (Mac) or
```

Comment:
For some strange reason, command + shift + I tries to open Apple Mail. Other users might find this too? It's a nit, but thought I'd mention it.